goalie/go_mesh_seq.py

```diff
@@ -228,6 +233,29 @@ def indicate_errors(
                     indi.interpolate(abs(indi))
                     self.indicators[f][i][j].interpolate(ufl.max_value(indi, 1.0e-16))
 
+            # discard current subinterval duplicate solution fields
+            if not self.steady:
+                for f in self.fields:
+                    self.solutions[f][FWD_OLD].pop(-1)
+                    self.solutions[f][ADJ_NEXT].pop(-1)
+                    enriched_mesh_seq.solutions[f][FWD_OLD].pop(-1)
+                    enriched_mesh_seq.solutions[f][ADJ_NEXT].pop(-1)
+
+            # delete current subinterval enriched mesh to reduce the memory footprint
+            enriched_mesh_seq.meshes.pop(-1)
+            enriched_mesh_seq.time_partition.drop_last_subinterval()
+            enriched_mesh_seq._update_function_spaces()
+
+        # clear empty labels
+        for f in self.fields:
+            if self.steady:
+                self.solutions.labels = "forward"
```

Comment:
Does this need to be inside a tuple?

Reply:
@jwallwork23 see commit https://github.com/mesh-adaptation/goalie/commit/1b358796ff12b49933b2a28ad150d0f570787a37 - the updated `label` variable now inside a tuple. As it only contains one item, I also added a comma such that it will be parsed correctly when used as an iterator value. Do you know of a better way to do this?